GUI/CalibrateTaskWindow.xaml

```diff
@@ -5,8 +5,20 @@
         xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
         xmlns:local="clr-namespace:MetaMorpheusGUI"
         mc:Ignorable="d"
+        xmlns:sys="clr-namespace:System;assembly=mscorlib"
+        xmlns:mzl="clr-namespace:MassSpectrometry;assembly=MassSpectrometry"
         KeyDown="KeyPressed"
         Title="Calibrate Task" Width="750" Height="750"  WindowStartupLocation="CenterScreen">
+
```

Comment:
I think the positioning of the pull-down menu needs to be adjusted. Also make sure we don't offer custom dissociation yet.